packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts

```diff
@@ -13,7 +13,7 @@ import { ITokenClaims } from "@fluidframework/protocol-definitions";
 import { KJUR as jsrsasign } from "jsrsasign";
 import { v4 as uuid } from "uuid";
 
-export const defaultTinyliciousPort = 35843;
+export const defaultTinyliciousPort = 3000;
```

Comment:
I think we should still pick a default other than 3000, since that has known conflicts with popular tooling (create react app). See #5415 for more details.